src/components/ae-splash/ae-splash.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="ae-splash" :class="{ [fill]: Boolean(fill) }">
+    <!-- @slot Default slot -->
+    <slot />
+  </div>
+</template>
+<script>
+export default {
+  name: 'ae-splash',
+  props: {
+    /**
+     * Fill property changes the color state of the splash screen
+     * select something between: `primary, secondary, neutral, alternative`
+     */
+    fill: {
+      type: String,
+      validator: value => [
+        'primary',
+        'secondary',
+        'neutral',
+        'alternative',
+      ].includes(value),
+    },
+  },
+};
+</script>
+<style lang="scss" scoped>
+@import "../../styles/variables/colors";
+
+.ae-splash {
+  display: flex;
+  flex-direction: column;
```

Comment:
Is it necessary to use flexbox in this component? Maybe somebody who uses this component should decide?